arangosh/Restore/RestoreFeature.cpp

```diff
@@ -359,6 +376,16 @@ void checkEncryption(arangodb::ManagedDirectory& directory) {
   }
 }
 
+void getDBProperties(arangodb::ManagedDirectory& directory, VPackBuilder& builder) {
+  try {
+    VPackBuilder fileContentBuilder = directory.vpackFromJsonFile("dump.json");
+    builder.add(fileContentBuilder.slice().get(arangodb::StaticStrings::Properties));
```

Comment:
what happens if there are no properties stored in the JSON file? At least we need to handle this gracefully.